### PR TITLE
Changed max to argmax

### PR DESCRIPTION
**Pull request by Chipe1**
Changed max() to argmax() to make code easier to read.

### CODE REVIEW

File: text.py [hunk 1]
```diff
@@ -380,8 +378,9 @@ def __init__(self, initial=None, goal=None, decoder=None):
 
     def actions(self, state):
         # Find the best
-        p, plainchar = max([(self.decoder.P1[c], c)
-                            for c in alphabet if c not in state])
+                            
+        search_list = [c for c in alphabet if c not in state]
+        plainchar = argmax(search_list, key=lambda c: self.decoder.P1[c])
```
Comment (norvig) [inline text.py]: This is fine, but I wonder which is clearer:
     lambda c: self.decoder.P1[c]
or
     self.decoder.P1.get

**Comment by norvig** [comment]
Sorry, I was late at getting to this, and now there is a conflict; can you fix?

File: text.py (hunk 1 above)
Comment (Chipe1) [inline text.py]: Did you mean self.decoder.P1.__getitem__
I got an error when I tried self.decoder.P1.get `AttributeError: 'UnigramTextModel' object has no attribute 'get'`

**Comment by Chipe1** [comment]
Updated PR in #481

**Pull request closed without merging**